package.json

```diff
@@ -106,6 +106,7 @@
     "@luma.gl/shadertools": "^8.1.0",
     "fast-xml-parser": "^3.16.0",
     "geotiff": "^1.0.0-beta.11",
+    "quickselect": "^2.0.0",
     "zarr": "^0.2.3"
```

Comment:
Need to make sure the latest version of zarr. I just released `0.3.0` (not beta) so please update to that!

Reply:
Sorry, yes, just. noticed the merge conflict.